storage/innobase/xtrabackup/src/read_filt.cc

```diff
@@ -245,7 +248,12 @@ static void rf_page_tracking_get_next_batch(xb_fil_cur_t *cursor,
    * since the last backup. Hence we copy all changes to mysql.ibd since last
    * backup start_lsn instead of last backup end_lsn.
    * We read all pages by setting read_batch_len to the size of file */
-  if (ctxt->space_id == dict_sys_t::s_dict_space_id) {
+
+  /* if DDL happened on the table after the checkpoint LSN we do full scan on
```

Comment:
Not all DDL, can we be more specific here that is only MLOG_INDEX_LOAD. Also, add a note about `s_dict_space_id`

Reply:
added inplace DDL. note about s_dict_space_id is mentioned above.